### PR TITLE
[[ Feature fontmap ]] Add the possibility to map font names to PostScript names.

### PR DESCRIPTION
**Pull request by livecodesebastien**
A default file, mapping all the Mac fonts display name to their respective PostScript, had been added.
Users can add their own mapping file, as a menu has been added in the iOS standalone settings menu.

This pull request needs to have its equivalent runrev/feature-ios_fontmap merged in livecode-ide

### CODE REVIEW

**Comment by runrevmark** [comment]
Reviewed-by: @runrevmark.

File: engine/rsrc/fontmap [hunk 1]
```diff
@@ -0,0 +1 @@
...
Yuanti SC Regular=STYuanti-SC-Regular
YuGothic Bold=YuGo-Bold
YuGothic Medium=YuGo-Medium
YuMincho Demibold=YuMin-Demibold
YuMincho Medium=YuMin-Medium
Yuppy SC Regular=YuppySC-Regular
Yuppy TC Regular=YuppyTC-Regular
Zapf Dingbats=ZapfDingbatsITC
Zapfino=Zapfino
Zawgyi-One=Zawgyi-OnePalatino Italic=Palatino-Italic
```
Comment (runrevmark) [inline engine/rsrc/fontmap]: I think there are line-ending issues with this file... Also there is a missing return after Zawgyi-OnePaletino

**Comment by livecodesebastien** [comment]
Pull request updated.

**Comment by runrevmark** [comment]
Reviewed-by: @runrevmark.

**Pull request merged**